packages/synapse-interface/constants/tokens/bridgeable.ts

```diff
@@ -1269,3 +1270,20 @@ export const WLD = new Token({
   priorityRank: 106,
   routeSymbol: 'WLD',
 })
+
+export const BNB = new Token({
+  priorityRank: 300,
+  visibilityRank: 300,
+  addresses: {
+    [CHAINS.BNB.id]: '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE',
```

Comment:
Use `zeroAddress` here as we haven't converted these over to `0xEee..` addresses on fe.